src/OrganisationRegistry.UI/app/organisation/management/overview/overview.template.html

```diff
@@ -1,11 +1,18 @@
 <div class="label-card__content-wrapper">
-  <div class="cta-title">
-      <h3 class="h3 cta-title__title">Stopzetting in de KBO</h3>
+  <div class="contact-card__content-wrapper">
+    <div class="cta-title">
+      <h3 class="h3 cta-title__title">Status in de KBO</h3>
+    </div>
+    <p class='ww-description'>
+      <span>
+        Hier vindt u de status van deze organisatie in de KBO (Kruispuntbank Ondernemingen) indien gekoppeld met KBO.
```

Comment:
should this be `        Hier vindt u de status van deze organisatie in de KBO (Kruispuntbank Ondernemingen) indien gekoppeld met de KBO.`?